### PR TITLE
fix(sk): fix stochastic_muzero_model_mlp.py with chance encoder

### PR DESCRIPTION
**Pull request by ShivamKumar2002**
Fixes issues mentioned in #283

### CODE REVIEW

File: zoo/classic_control/cartpole/envs/cartpole_lightzero_env.py [hunk 1]
```diff
@@ -87,6 +94,10 @@ def reset(self) -> Dict[str, np.ndarray]:
 
         action_mask = np.ones(self.action_space.n, 'int8')
         obs = {'observation': obs, 'action_mask': action_mask, 'to_play': -1}
+        if self._enable_chance:
+            chance_value = random.randint(0, 2)
+            obs['observation'][chance_value] *= 2
```
Comment (puyuan1996) [inline zoo/classic_control/cartpole/envs/cartpole_lightzero_env.py:99]: Add a comment like this: This is to artificially introduce randomness in order to evaluate the performance of stochastic_muzero on state input.

Comment (ShivamKumar2002) [reply zoo/classic_control/cartpole/envs/cartpole_lightzero_env.py:102]: added comment

File: zoo/classic_control/cartpole/config/cartpole_stochastic_muzero_config.py [hunk 2]
```diff
@@ -11,6 +11,7 @@
 batch_size = 256
 max_env_step = int(1e5)
 reanalyze_ratio = 0
+enable_chance = False
```
[the remark this answers is not in the document]
Comment (ShivamKumar2002) [reply zoo/classic_control/cartpole/config/cartpole_stochastic_muzero_config.py:14]: image

Here is the training graph with and without chance. Looks correct to me.

File: zoo/classic_control/cartpole/envs/cartpole_lightzero_env.py (hunk 1 above)
Comment (puyuan1996) [reply zoo/classic_control/cartpole/envs/cartpole_lightzero_env.py:102]: Thanks for your contribution!

**Pull request merged**